src/librustc/middle/expr_use_visitor.rs

```diff
@@ -828,7 +828,7 @@ impl<'a, 'gcx, 'tcx> ExprUseVisitor<'a, 'gcx, 'tcx> {
                 match bm {
                     ty::BindByReference(..) =>
                         mode.lub(BorrowingMatch),
-                    ty::BindByValue(..) => {
+                    ty::BindByValue{..} => {
```

Comment:
```suggestion
                    ty::BindByValue { .. } => {
```
(here and elsewhere)